package.json

```diff
@@ -18,21 +18,21 @@
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.4",
     "envy": "^2.0.0",
-    "expo": "^41.0.0",
+    "expo": "^42.0.0",
     "expo-app-loading": "^1.0.3",
-    "expo-font": "~9.1.0",
-    "expo-status-bar": "^1.0.4",
+    "expo-font": "~9.2.1",
+    "expo-status-bar": "~1.0.4",
     "history": "^5.0.0",
     "immer": "^9.0.1",
     "normalizr": "^3.6.1",
     "prop-types": "^15.7.2",
-    "react": "^17.0.2",
```

Comment:
Expo officially supports React 16, and most plugins list it as a peer dependency. So for now let's just use React 16.

Reply:
Broooonnnnnkyyyyyyyy!!!!